backend/pkg/modules/internal/bfla/restapi/openapi.yaml

```diff
@@ -19,6 +19,28 @@ paths:
               schema:
                 $ref: '../../../../../../api3/common/openapi.yaml#/components/schemas/ModuleVersion'
 
+  /api/findings/{apiID}:
+    get:
+      operationId: GetAPIFindingsForAPI
+      summary: 'Get Findings for an API'
+      description: 'Asks APIClarity for all findings of all modules of a specific API. Implemented at the core of APIClarity.'
+      parameters:
+        - $ref: '../../../../../../api3/common/openapi.yaml#/components/parameters/ApiID'
```

Comment:
As per my previous comment, please use the following:

```
      - name: apiID
          in: path
          required: true
          schema:
            $ref: ../../../../../../api3/common/openapi.yaml#/components/schemas/ApiID
```

Reply:
fixed